File: solutions/2977-check-if-a-string-is-an-acronym-of-words/solution.py
class Solution(object):
    def isAcronym(self, words, s):
        """
        :type words: List[str]
        :type s: str
        :rtype: bool
        """
        if len(s)!=len(words):
            return False
        for i in range(0,len(s)):
            if words[i][0]!=s[i]:
                return False
        return True

File: solutions/2977-check-if-a-string-is-an-acronym-of-words/solution.java
class Solution {
    public boolean isAcronym(List<String> words, String s) {
        int i=0;
        if(words.size()!=s.length())
            return false;
        for(String a:words)
        {
            if(a.charAt(0)!=s.charAt(i))
                return false;
            i++;
        }
        return true;
    }
}
